README.md

```diff
@@ -1 +1,31 @@
 # ResetPasswordBundle
+
+Worrying about how to deal with users that can't remember their password? We've got you covered! This bundle provides a secure out of the box solution to allow users to reset their forgotten passwords.
+
+## Requirements
+
+Most Symfony application's using Symfony 4.4 or newer should already have all the required dependencies to use this bundle, and the optional traits that are included with it. 
+
+If your application isn't a traditional Symfony web application, see the [detailed requirements](https://github.com/SymfonyCasts/reset-password-bundle/wiki/Dependencies) in our wiki.
```

Comment:
I'm not sure this section - or the WIKI - is necessary. Composer will obviously enforce the required dependencies - like `symfony/config`, etc. For the optional once, like `doctrine/orm` to use the trait, I guess you wouldn't even try to use this unless you had a Doctrine repository? Anyways, I feel like this adds noise - but for 99% of users, I think it may make them just more confused.

Reply:
I was questioning this myself. Keep it in place for now, to be removed before release..

I think we do need to mention Symfony 4.4+ somewhere. I'd have to go back and double check, but I believe this can't be used in previous versions. Probably fit that in up in the description..